cli/pkg/local/server.go

```diff
@@ -561,7 +583,22 @@ func (s *Server) RedeployProject(ctx context.Context, r *connect.Request[localv1
 		}
 		defer release()
 
-		assetID, err := cmdutil.UploadRepo(ctx, repo, s.app.ch, r.Msg.Org, r.Msg.ProjectName)
+		var projName, orgName string
+		if r.Msg.ProjectId != "" {
+			proj, err := c.GetProjectByID(ctx, &adminv1.GetProjectByIDRequest{
+				Id: projName,
```

Comment:
This should be `r.Msg.ProjectId` and not `projName` ?

Reply:
Ah good catch